### PR TITLE
commented out unused decalaration

### PR DESCRIPTION
**Pull request by kdriver**
Closes #

## Proposed Changes

Comment out/remove line 204 in FluxParser - unused declaration

## Checklist

- [ ] CHANGELOG.md updated
- [ ] Rebased/mergeable
- [ ] A test has been added if appropriate
- [ ] Tests pass
- [ ] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)

### CODE REVIEW

File: src/query/FluxParser.cpp [hunk 1]
```diff
@@ -201,7 +201,7 @@ bool FluxQueryResult::next() {
 }
 
 FluxDateTime *FluxQueryResult::convertRfc3339(String value, const char *type) {
-    FluxDateTime *ft = nullptr;
+    // unused declaration FluxDateTime *ft = nullptr;
```
Comment (vlastahajek) [inline src/query/FluxParser.cpp:204]: Why just commenting? Remove this, please.

Comment (vlastahajek) [inline src/query/FluxParser.cpp:204]: Why just commenting? Remove this, please.

**Review by vlastahajek — CHANGES_REQUESTED** [review]

File: src/query/FluxParser.cpp (hunk 1 above)
Comment (kdriver) [reply src/query/FluxParser.cpp]: OK, have removed the line and comitted back in on the branch

**Comment by vlastahajek** [comment]
Thanks for the PR!

**Pull request merged**